flax/linen/normalization.py

```diff
@@ -335,6 +343,70 @@ def __call__(self, x):
         self.bias_init, self.scale_init)
 
 
+class RMSNorm(Module):
```

Comment:
Could you please add an example of how to use the layer? I think we should starting doing this for every layer, like @cgarciae does in his RNN PR: https://github.com/google/flax/pull/2604/files#r1107264719.